content/applications/hr/recruitment/new_job.rst

```diff
@@ -99,27 +108,32 @@ An *Interview Form* is used to determine if a candidate is a good fit for a job
 forms can be as specific or general as desired, and can take the form of a certification, an exam,
 or a general questionnaire. Interview forms are determined by the recruitment team.
 
-All interview forms must be created, there are no pre-configured forms in Odoo. To create an
-interview form, start from the recruitment tab of the :guilabel:`Job Position` form. In the
-:guilabel:`Interview Form` field, enter a name for the new interview form. As the name is typed,
-several options populate beneath the entry, :guilabel:`Create (interview form name)`,
-:guilabel:`View all`, and :guilabel:`Create and edit...`. Click :guilabel:`Create and edit...`. and
-a :guilabel:`Create Interview Form` pop-up window appears.
+Before creating an interview form, ensure the proper settings are enabled. Navigate to
+:menuselection:`Recruitment app --> Configuration --> Settings`, and under the
+:guilabel:`Recruitment Process` section, ensure the :guilabel:`Send Interview Survey` option is
+enabled.
+
+Since there are no pre-configured forms in Odoo, all interview forms must be created. To create an
+interview form, start from the :guilabel:`Recruitment` tab of the :guilabel:`Job Position` form. In
+the :guilabel:`Interview Form` field, enter a name for the new interview form. As the name is typed,
+several options populate beneath the entry: :guilabel:`Create (interview form name)`,
+:guilabel:`Search More...`, and :guilabel:`Create and edit...`. Click :guilabel:`Create and edit...`
+and a :guilabel:`Create Interview Form` pop-up window appears.
```

Comment:
```suggestion
Since there are no pre-configured forms in Odoo, all interview forms must be created. 

To create an interview form, start from the :guilabel:`Recruitment` tab of the :guilabel:`Job Position` form. In
the :guilabel:`Interview Form` field, enter a name for the new interview form. As the name is typed,
several options populate beneath the entry: :guilabel:`Create (interview form name)`,
:guilabel:`Search More...`, and :guilabel:`Create and edit...`. Click :guilabel:`Create and edit...`
and a :guilabel:`Create Interview Form` pop-up window appears.
```

Reply:
an image of the pop-up would be nice here